src/old-components/Guilds/AddressButton/index.tsx

```diff
@@ -2,7 +2,7 @@ import { isDesktop } from 'react-device-detect';
 import styled from 'styled-components';
 import React from 'react';
 import { IconButton } from '../common/Button';
-import useENSAvatar from '../../../hooks/Guilds/ether-swr/ens/useENSAvatar';
+import useENSAvatar from '../../../hooks/Guilds/ens/useENSAvatar';

```

Comment:
```suggestion
import useENSAvatar from 'hooks/Guilds/ens/useENSAvatar';
```
Let's try using absolute path import